src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenario.kt

```diff
@@ -20,8 +20,13 @@ class JiraCoreScenario constructor() : Scenario {
 
     override fun getActions(jira: WebJira, seededRandom: SeededRandom, meter: ActionMeter): List<Action> {
         initializeIssueKeyMemory(seededRandom)
-        val projectMemory = AdaptiveProjectMemory(random = seededRandom)
         val jqlMemory = AdaptiveJqlMemory(seededRandom)
+        val projectMemory = JqlRememberingProjectMemory
```

Comment:
One thing that I'm worried about is that with this the JQL memory will be flooded with "project = " queries. I guess the complicated logic inside the `AdaptiveJqlMemory` is what is supposed to prevent that, however the `JqlRememberingProjectMemory` ommits it.

Reply:
I added the `LimitedJqlMemory` to address that. See CHANGELOG.md and it's code to understand what it actually is and please make any suggetions if it could be better in any way.